src/hatti/views/table.cljs

```diff
@@ -121,14 +120,19 @@
   ([form get-label? language & {:keys [is-filtered-dataview? owner]}]
    (let [columns (for [field (all-fields form :is-filtered-dataview?
                                          is-filtered-dataview?)]
-                   (let [{:keys [name type full-name]} field
+                   (let [{:keys [name type full-name], {:keys [hxl]} :instance} field
                          label (if get-label? (get-label field language) name)]
-                     {:id name :field full-name :type type
-                      :name label :toolTip label :sortable true
+                     {:id name
+                      :field full-name
+                      :type type
+                      :name (if hxl (str label "<div class=\"hxl-row\">" hxl "</div>") label)
```

Comment:
using the class `hxl-row` doesn't feel semantic because it's specific to the particular HXL coding, HXL is a type of coding, `coding-row` may be a better candidate to describe the meaning of this class. @okal any thoughts?